bims/api_views/send_notification_to_validator.py

```diff
@@ -0,0 +1,39 @@
+# coding=utf-8
+from braces.views import LoginRequiredMixin
+from django.core.mail import send_mail
+from django.http import JsonResponse, HttpResponse
+from django.contrib.auth.models import Group
+from rest_framework.views import APIView
+from rest_framework import status
+from geonode.people.models import Profile
+
+
+class SendNotificationValidation(LoginRequiredMixin, APIView):
+
+    def get(self, request):
+        site = request.get_host()
+        scheme = request.scheme
+        pk = request.GET.get('pk', None)
+        group = Group.objects.get(name='validator')
```

Comment:
don't use this permission check, try to do something like this https://github.com/kartoza/django-bims/blob/develop/bims/views/csv_upload.py#L40

Reply:
https://github.com/kartoza/django-bims/blob/develop/bims/models/biological_collection_record.py#L85